.circleci/config.yml

```diff
@@ -111,14 +111,18 @@ jobs:
       - run:
           name: Test - Clone planet4-docker-compose
           command: |
-            git clone https://github.com/greenpeace/planet4-docker-compose
+            git clone -b planet-5077 https://github.com/greenpeace/planet4-docker-compose
       - run:
-          name: Test - Run tests
+          name: Test - Run Codeception tests
           command: |
             export BUILD_TAG="build-$(cat /tmp/workspace/var/circle-build-num)"
             export APP_IMAGE=gcr.io/planet-4-151612/planet4-base-app:${BUILD_TAG}
             export OPENRESTY_IMAGE=gcr.io/planet-4-151612/planet4-base-openresty:${BUILD_TAG}
             make -C planet4-docker-compose ci test
+      - run:
+          name: Test - Run E2E Jest/Puppeteer tests
+          command: |
+            make -C planet4-docker-compose jstests
```

Comment:
If there is a lot happening in a make target, it can be handy to include the sub-make targets (like `installnpm installdeps assets runjstests
` here) as separate steps. That will give us the duration of each part, and also make the logs a lot more manageable (Less need to scroll through large amount of console output to find relevant piece.).

Maybe here not each one of `installnpm installdeps assets runjstests` needs a separate step, but definitely for the one that actually runs the test this would be nice.